### PR TITLE
Update terminologies - Closes 316

### LINKED ISSUE #316: Update terminology in the Hello frontend and boilerplate
Update to latest terminology in the Hello frontend and boilerplate.

### PR DESCRIPTION
**Pull request by TalhaMaliktz**
Closes #316

### CODE REVIEW

File: guides/07-ui-boilerplate/hello_frontend/src/components/home.js [hunk 1]
```diff
@@ -11,11 +11,11 @@ function Home() {
             <FixedMenuLayout />
             <Container >
                 <span>
-                    <h1><img src={logo} alt="Lisk Logo" width="35" style={{ display: 'inline', verticalAlign: 'middle' }} /> Welcome to the Hello sidechain application!</h1>
+                    <h1><img src={logo} alt="Lisk Logo" width="35" style={{ display: 'inline', verticalAlign: 'middle' }} /> Welcome to the Hello Lisk application!</h1>
                 </span>
                 <Divider>
                 </Divider>
-                <h4>The Hello sidechain application allows you to do the following: </h4>
+                <h4>The Hello application allows you to do the following: </h4>
```
Comment (TalhaMaliktz) [inline guides/07-ui-boilerplate/hello_frontend/src/components/home.js]:
```suggestion
                <h4>The Hello Lisk application allows you to do the following: </h4>
```

File: guides/chain-registration/sidechain/aggregatedBFTWeight.js [hunk 2]
```diff
@@ -1,7 +1,7 @@
 const { apiClient } = require('@liskhq/lisk-client');
 
 (async () => {
-	// Update the path to point to your sidechain app data folder
+	// Update the path to point to your Lisk app's data folder
```
Comment (Tschakki) [inline guides/chain-registration/sidechain/aggregatedBFTWeight.js]:
```suggestion
	// Update the path to point to your blockchain client's data folder
```

File: guides/chain-registration/sidechain/aggregatedBFTWeight.js [hunk 3]
```diff
@@ -1,7 +1,7 @@
...
 	const sidechainClient = await apiClient.createIPCClient('~/.lisk/hello_client');
 	const sidechainNodeInfo = await sidechainClient.invoke('system_getNodeInfo');
```
Comment (Tschakki) [inline guides/chain-registration/sidechain/aggregatedBFTWeight.js:6]: Please update the variable names, too, where needed.

```suggestion
	const blockchainClient = await apiClient.createIPCClient('~/.lisk/hello_client');
	const blockchainNodeInfo = await blockchainClient.invoke('system_getNodeInfo');
```

**Review by Tschakki — CHANGES_REQUESTED** [review]

File: tutorials/hello/hello_client/src/app/plugins/hello_info/hello_info_plugin.ts [hunk 4]
```diff
@@ -40,7 +40,7 @@ export class HelloInfoPlugin extends BasePlugin<HelloInfoPluginConfig> {
 
 	// Syncs on-chain event's data with plugin's database.
 	private async _syncChainEvents(): Promise<void> {
-		// 1. Get latest block height from the sidechain
+		// 1. Get latest block height from the blockchain
```
Comment (chris529) [inline tutorials/hello/hello_client/src/app/plugins/hello_info/hello_info_plugin.ts]: It should read as follows: "1. Get the latest block height from the blockchain"

**Review by chris529 — APPROVED** [review]
LGTM! 👍

**Review by Tschakki — APPROVED** [review]
LGTM

File: guides/chain-registration/mainchain/aggregatedBFTWeight.js [hunk 5]
```diff
@@ -3,23 +3,23 @@ const { apiClient } = require('@liskhq/lisk-client');
...
-	const mainchainClient = await apiClient.createWSClient(RPC_ENDPOINT);
-	const mainchainNodeInfo = await mainchainClient.invoke('system_getNodeInfo');
+	const blockchainClient = await apiClient.createWSClient(RPC_ENDPOINT);
+	const mainchainNodeInfo = await blockchainClient.invoke('system_getNodeInfo');
 	// Get active validators from mainchain
 	const {
 		validators: mainchainActiveValidators,
 		certificateThreshold: mainchainCertificateThreshold
-	} = await mainchainClient.invoke('consensus_getBFTParameters', { height: mainchainNodeInfo.height });
+	} = await blockchainClient.invoke('consensus_getBFTParameters', { height: mainchainNodeInfo.height });
```
Comment (Tschakki) [inline guides/chain-registration/mainchain/aggregatedBFTWeight.js]: Please revert the changes

File: guides/chain-registration/mainchain/paramsCreation.ts [hunk 6]
```diff
@@ -12,14 +12,14 @@ const SC_NAME = "sidechain1";
...
-	const mainchainNodeInfo = await mainchainClient.invoke('system_getNodeInfo');
+	const blockchainClient = await apiClient.createIPCClient('~/.lisk/lisk-core');
+	const mainchainNodeInfo = await blockchainClient.invoke('system_getNodeInfo');
 
 	// Get active validators from mainchain
 	const {
 		validators: mainchainActiveValidators,
 		certificateThreshold: mainchainCertificateThreshold
-	} = await mainchainClient.invoke('consensus_getBFTParameters', { height: mainchainNodeInfo.height });
+	} = await blockchainClient.invoke('consensus_getBFTParameters', { height: mainchainNodeInfo.height });
```
Comment (Tschakki) [inline guides/chain-registration/mainchain/paramsCreation.ts]: Please revert the changes

File: guides/chain-registration/sidechain/aggregatedBFTWeight.js [hunk 7]
```diff
@@ -1,11 +1,11 @@
 const { apiClient } = require('@liskhq/lisk-client');
 
 (async () => {
-	// Update the path to point to your sidechain app data folder
-	const sidechainClient = await apiClient.createIPCClient('~/.lisk/hello_client');
-	const sidechainNodeInfo = await sidechainClient.invoke('system_getNodeInfo');
+	// Update the path to point to your blockchain client's data folder
+	const blockchainClient = await apiClient.createIPCClient('~/.lisk/hello_client');
+	const sidechainNodeInfo = await blockchainClient.invoke('system_getNodeInfo');
```
Comment (Tschakki) [inline guides/chain-registration/sidechain/aggregatedBFTWeight.js]:
```suggestion
	// Update the path to point to your sidechain client's data folder
	const sidechainClient = await apiClient.createIPCClient('~/.lisk/hello_client');
	const sidechainNodeInfo = await sidechainClient.invoke('system_getNodeInfo');
```

File: guides/chain-registration/sidechain/aggregatedBFTWeight.js [hunk 8]
```diff
@@ -1,11 +1,11 @@
...
 	// Get active validators from sidechain
-	const bftParams = await sidechainClient.invoke('consensus_getBFTParameters', { height: sidechainNodeInfo.height });
+	const bftParams = await blockchainClient.invoke('consensus_getBFTParameters', { height: sidechainNodeInfo.height });
```
Comment (Tschakki) [inline guides/chain-registration/sidechain/aggregatedBFTWeight.js]:
```suggestion
	const bftParams = await sidechainClient.invoke('consensus_getBFTParameters', { height: sidechainNodeInfo.height });
```

File: guides/chain-registration/sidechain/sidechainRegistration.ts [hunk 9]
```diff
@@ -25,26 +25,26 @@ const { apiClient } = require('@liskhq/lisk-client');
         }
     });
 
-    const sidechainClient = await apiClient.createIPCClient('~/.lisk/sidechain');
+    const blockchainClient = await apiClient.createIPCClient('~/.lisk/sidechain');
 
-    const sidechainNodeInfo = await sidechainClient.invoke('system_getNodeInfo');
+    const sidechainNodeInfo = await blockchainClient.invoke('system_getNodeInfo');
```
Comment (Tschakki) [inline guides/chain-registration/sidechain/sidechainRegistration.ts]: Please revert the changes

File: guides/chain-registration/sidechain/sidechainRegistration.ts [hunk 10]
```diff
@@ -25,26 +25,26 @@ const { apiClient } = require('@liskhq/lisk-client');
...
 
     // Get active validators from sidechain
-    const { validators: sidehcainActiveValidators } = await sidechainClient.invoke('consensus_getBFTParameters', { height: sidechainNodeInfo.height });
+    const { validators: sidechainActiveValidators } = await blockchainClient.invoke('consensus_getBFTParameters', { height: sidechainNodeInfo.height });
```
Comment (Tschakki) [inline guides/chain-registration/sidechain/sidechainRegistration.ts]:
```suggestion
    const { validators: sidechainActiveValidators } = await sidechainClient.invoke('consensus_getBFTParameters', { height: sidechainNodeInfo.height });
```

**Review by Tschakki — CHANGES_REQUESTED** [review]
Sorry I overlooked that some changes were not regarding the boilerplate and Hello frontend. Please revert the corresponding changes, they are out of scope of this issue.

File: guides/chain-registration/sidechain/aggregatedBFTWeight.js [hunk 11]
```diff
@@ -1,9 +1,9 @@
 const { apiClient } = require('@liskhq/lisk-client');
 
 (async () => {
-	// Update the path to point to your sidechain app data folder
-	const sidechainClient = await apiClient.createIPCClient('~/.lisk/hello_client');
-	const sidechainNodeInfo = await sidechainClient.invoke('system_getNodeInfo');
+	// Update the path to point to your sidechain client's data folder
+	const blockchainClient = await apiClient.createIPCClient('~/.lisk/hello_client');
+	const blockchainNodeInfo = await blockchainClient.invoke('system_getNodeInfo');
```
Comment (TalhaMaliktz) [inline guides/chain-registration/sidechain/aggregatedBFTWeight.js]:
```suggestion
	const sidechainClient = await apiClient.createIPCClient('~/.lisk/hello_client');
	const sidechainNodeInfo = await sidechainClient.invoke('system_getNodeInfo');
```

File: guides/chain-registration/sidechain/aggregatedBFTWeight.js (hunk 3 above)
Comment (TalhaMaliktz) [reply guides/chain-registration/sidechain/aggregatedBFTWeight.js:6]: The code changes suggested in this comment were incorporated into the original code, but I have now restored the code to its original state based on your latest suggestion.

**Review by Tschakki — APPROVED** [review]
Thank you! 👍
